xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -828,6 +850,28 @@ AVDictionary* CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput()
         av_dict_set(&options, "cookies", cookies.c_str(), 0);
     }
   }
+  else if (url.IsProtocol("udp") || url.IsProtocol("rtp"))
+  {
+    std::string strURL = url.Get();
+    size_t found = strURL.find("://");
+    if (found != std::string::npos)
+    {
+      size_t start = found + 3;
+      size_t found = strURL.find("@");
+      if (found != std::string::npos && found > start)
+      {
+        // sourceip found
+        std::string strSourceIp = strURL.substr(start, found - start);
+        CLog::Log(LOGDEBUG,
+                  "CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput() UDP/RTP sources '%s'",
+                  strSourceIp.c_str());
+        av_dict_set(&options, "sources", strSourceIp.c_str(), 0);
+        av_dict_set(&options, "connect", "1", 0);
+        av_dict_set(&options, "write_to_source", "1", 0);
+        //av_dict_set(&options, "buffer_size", "200000", 0);
```

Comment:
If this is not needed, please remove the line.

Reply:
Will do @Rechi, as the original fix did not work entirely I've added a WIP tag.